manifests/nodejs.yml

```
@@ -7,16 +7,16 @@ tests/:
   appsec/:
     api_security/:
       test_schemas.py:
-        Test_Scanners: missing_feature
-        Test_Schema_Request_Cookies: missing_feature
-        Test_Schema_Request_FormUrlEncoded_Body: missing_feature
-        Test_Schema_Request_Headers: missing_feature
-        Test_Schema_Request_Json_Body: missing_feature
-        Test_Schema_Request_Path_Parameters: missing_feature
-        Test_Schema_Request_Query_Parameters: missing_feature
+        Test_Scanners: v4.21.0
+        Test_Schema_Request_Cookies: v4.21.0
+        Test_Schema_Request_FormUrlEncoded_Body: v4.21.0
+        Test_Schema_Request_Headers: v4.21.0
+        Test_Schema_Request_Json_Body: missing_feature # v4.21.0
```

this should be supported but it fails ?
```suggestion
        Test_Schema_Request_Json_Body: v4.21.0
```